src/util.py

```diff
@@ -260,12 +265,89 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def set_vector_density(data, minDistance=0, otherVars=[]):
+    """
+    Utility function to change density of vector plots.
+
+    Ars:
+
+        data (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains the vector plot latitude/longitude data.
+
+        minDistance (:class:`int`):
+            Value in degrees that determines the distance between the vectors.
+
+        otherVars (:class: array of `xarray.core.dataarray.DataArray`):
+            Other variables that require the same density reduction as the lat/lon data
+
+    """
+    import math
+    import warnings
+
+    if minDistance < 0:
+        raise Exception("minDistance cannot be negative.")
+
+    # Change the density with parameter "minDistance"
+    if minDistance != 0:
```

Comment:
If `minDistance == 0` then `ds` would be returned immediately without being initialized. It shouldn't be the case I believe. Please check the entire data flow throughout the code.